astropy/visualization/wcsaxes/core.py

```diff
@@ -596,6 +596,18 @@ def get_ylabel(self):
                 return coord.get_axislabel()
 
     def get_coords_overlay(self, frame, coord_meta=None):
+        """
+        Parameters
+        ----------
+        frame : str, `~astropy.coordinates.BaseCoordinateFrame`
+            Frame to get overlay for. If a string must correspond to
+            one of the coordinate frames registered in the astropy
+            frame transform graph.
+
+        Returns
+        -------
+        overlay : `~astropy.visualization.wcsaxes.CoordinatesMap`
```

Comment:
Should have description... but don't just accept my suggestion. Fill in with actual info. Thanks!
```suggestion
        overlay : `~astropy.visualization.wcsaxes.CoordinatesMap`
            Overlay of blah blah blah.
```